cl/disclosures/tasks.py

```diff
@@ -402,10 +435,22 @@ def import_disclosure(self, data: Dict[str, Union[str, int, list]]) -> None:
     :param data: The disclosure information to process
     :return: None
     """
-
     from cl.disclosures.models import FinancialDisclosure
     from cl.people_db.models import Person
 
+    # Check download_filepath to see if it has been processed before.
+    if has_been_extracted(data):
+        logger.info(f"Document already extracted and saved: {data['id']}.")
+        return
+
+    interface = make_redis_interface("DISCLOSURES")
```

Comment:
use `CACHE` here, not `DISCLOSURES`. We don't need a full redis DB just for this (and if we did, we'd have to add it in our settings.)